README.md

```diff
@@ -45,7 +45,7 @@ Still, some restrictions are inevitable:
 ## A note on the limited area setting
 Currently we are using these models on a limited area covering the Nordic region, the so called MEPS area (see [paper](https://arxiv.org/abs/2309.17370)).
 There are still some parts of the code that is quite specific for the MEPS area use case.
-This is in particular true for the mesh graph creation (`create_mesh.py`) and some of the constants used (`neural_lam/constants.py`).
+This is in particular true for the mesh graph creation (`create_mesh.py`) and some of the constants used (`neural_lam/data_config.yaml`).
```

Comment:
I think it would be best if it is completely up to the user how and where the config is stored. Then you just give any path to the yaml-file as input to the main script.

Reply:
> I think it would be best if it is completely up to the user how and where the config is stored. Then you just give any path to the yaml-file as input to the main script.

yes, that was what I was thinking too :) I was just being a bit more long-winded about it. The README here suggests to me that it _should_ reside inside `neural_lam/`. Maybe we should change this to `data_config.yaml` instead rather than `neural_lam/data_config.yaml` then?

Reply:
Agreed, I changed the Readme accordingly.